packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -82,6 +83,10 @@ class SegmentedControl<T> extends StatefulWidget {
     @required this.children,
     @required this.onValueChanged,
     this.groupValue,
+    this.unselectedColor = CupertinoColors.white,
+    this.selectedColor = CupertinoColors.activeBlue,
+    this.borderColor = CupertinoColors.activeBlue,
+    this.pressedColor = const Color(0x33007AFF),
```

Comment:
add asserts to check that the new parameters are not null (or is it legal for them to be null)?